src/template-tags/tickets.php

```diff
@@ -748,11 +748,15 @@ function tribe_tickets_has_meta_fields( $id, $context = null ) {
 }
 
 /**
- * Updates a given Object Capacity
+ * Removes all meta for a given object capacity
+ * object can be a ticket, or an event/post with tickets
+ *
+ * Note, you can pass an event/post to this function and it will merrily change the meta values
+ * for the event - not fot the tickets!
```

Comment:
This part is confusing to me - what is the purpose here, did we store capacity on events at some point? Do we now?

Reply:
```suggestion
 * for the event - not for the tickets!
```

Reply:
doh 🤦‍♂️

Reply:
Intentional https://tribe.slack.com/archives/C0258NY1C/p1556119457000700

But needs more tests as per your note